redash/authentication/__init__.py

```diff
@@ -194,7 +194,7 @@ def jwt_token_load_user_from_request(request):
     if not valid_token:
         return None
 
-    email = payload[org_settings["auth_jwt_auth_user_claim"]]
+    email = payload["identity"]
```

Comment:
maybe a comment like "we always inject this ourselves"? in the context of the whole PR it's clear, but seeing it on its own first was mildly surprising

Reply:
I ended up changing it to an explicit return value rather than a side-effect mutation because that seemed cleaner.